.gitignore

```diff
@@ -3,12 +3,8 @@ dist
 web
 node_modules
 
-# config
-config.json
```

Comment:
This is a leftover from when we had a default config file in the repo that was named differently and users were expected to copy it to `config.json` to make edits.